httpe/example_httpe_test.go

```diff
@@ -51,7 +51,7 @@ func writeErr(w http.ResponseWriter, err error) {
 }
 
 func ExampleHandlerFuncE() {
-	handler := httpe.NewHandlerFunc(handle, writeErr)
+	handler := httpe.NewHandlerFunc(handle, httpe.WithErrWriterFunc(writeErr))
```

Comment:
consider simplifying this example or creating a simple verions

Reply:
I have re-ordered the functions so main() comes first and added a couple of comments. I will think about what a good example might look like. We should probably do an example PR as there are other parts of the repo that could do with examples too (and more in httpe). For now, I'll stick with this one example if you don't mind.